shelvery_tests/cloudformation-unittest.yaml

```diff
@@ -0,0 +1,49 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Resources:
+
+  RDSInstance:
+    Type: AWS::RDS::DBInstance
+    Properties: 
+      DBInstanceClass:  db.t3.small
+      DBInstanceIdentifier: shelvery-test-rds
+      Engine: postgres
+      MasterUsername: asdf9876lkjh4312
+      MasterUserPassword: azxcv0987poiu4321
+      AllocatedStorage: '100'
+
+  RDSClusterInstance:
+    Type: AWS::RDS::DBCluster
+    Properties:
+      Engine: aurora-postgresql
+      MasterUsername: asdf9876lkjh4312
+      MasterUserPassword: azxcv0987poiu4321
+      DBClusterIdentifier: shelvery-test-rds-cluster
+  
+  DocDBInstance:
+    Type: AWS::DocDB::DBCluster
+    Properties:
+      MasterUsername: asdf9876lkjh4312
+      MasterUserPassword: azxcv0987poiu4321
+      StorageEncrypted: false
+      DBClusterIdentifier: shelvery-test-docdb
+  
+  EC2AmiInstance:
+    Type: AWS::EC2::Instance
+    Properties:
+      ImageId: ami-07620139298af599e
```

Comment:
this could break if that ami was removed. I would use a cloudformation parameter to lookup an ami id from a ssm parameter as the parameter is updated with the latest ami version when they're released

```yaml
Parameters:
  Ami:
    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
    Default: /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2
    
Resources:
  EC2AmiInstance:
    Type: AWS::EC2::Instance
    Properties:
      ImageId:
        Ref: Ami

Reply:
Added parameter for Ami ID